### ISSUE TITLE
Question on the GraphConsis

### ISSUE THREAD

**Issue by zltyrzshy** [safe-graph/DGFraud-TF2#12]
When I tried to run GraphConsis, I found that the model always predicted all nodes as negative (normal nodes), resulting in AUC=0.5000, F1=score=0.0000. I tried to modify the parameters, mainly for the learning rate and epoch, and other parameters. Consistent with the paper, I am very confused. The parameters are as follows：
`parser.add_argument('--seed', type=int, default=42, help='random seed')
parser.add_argument('--epochs', type=int, default=5,help='number of epochs to train')
parser.add_argument('--batch_size', type=int, default=512, help='batch size')
parser.add_argument('--train_size', type=float, default=0.8,help='training set percentage')
parser.add_argument('--lr', type=float, default=0.1, help='learning rate')
parser.add_argument('--nhid', type=int, default=128, help='number of hidden units')
parser.add_argument('--sample_sizes', type=list, default=[10, 5],help='number of samples for each layer')
parser.add_argument('--identity_dim', type=int, default=32,help='dimension of context embedding')
parser.add_argument('--eps', type=float, default=0.001,help='consistency score threshold ε')
args = parser.parse_args()`
